rpc_jobs/rpc_openstack.yml

```diff
@@ -179,11 +179,17 @@
       - "ironic"
       - "swift"
     action:
+      - system
```

Comment:
As this stands right now, the 'system' action has no image settings, so it inherits the defaults which are hopelessly too low for the MNAIO test. You'll need to copy over the IMAGE, FLAVOR, REGIONS, FALL_BACK_REGIONS settings from the 'deploy' action.